types/sanctuary/index.d.ts

```diff
@@ -294,30 +295,67 @@ declare namespace Sanctuary {
     //  Parse
     parseDate(s: string): Maybe<Date>;
     parseFloat(s: string): Maybe<number>;
-    parseInt(p: Integer): (q: string) => Maybe<Integer>;
+    parseInt(radix: 2 |
+                    3 |
+                    4 |
+                    5 |
+                    6 |
+                    7 |
+                    8 |
+                    9 |
+                   10 |
+                   11 |
+                   12 |
+                   13 |
+                   14 |
+                   15 |
+                   16 |
+                   17 |
+                   18 |
+                   19 |
+                   20 |
+                   21 |
+                   22 |
+                   23 |
+                   24 |
+                   25 |
+                   26 |
+                   27 |
+                   28 |
+                   29 |
+                   30 |
+                   31 |
+                   32 |
+                   33 |
+                   34 |
+                   35 |
+                   36):
+            (s: string) => Maybe<Integer>;
     parseJson(p: Predicate<any>): (q: string) => Maybe<any>;
     //  RegExp
     regex(p: string): (q: string) => RegExp;
     regexEscape(s: string): string;
     test(pattern: RegExp): Predicate<string>;
     match(pattern: RegExp): (q: string) => Array<Maybe<MatchObj>>;
-    matchAll(pattern: RegExp): (q: string) => MatchObj[];
+    matchAll(pattern: RegExp): (q: string) => Array<MatchObj>;
```

Comment:
@vicrac, is it possible to define [`$.GlobalRegExp`][1] in TypeScript? I tried `RegExp & { global: true }`, but the `global` property of `/x/g` is not known to be of type `true`.


[1]: https://github.com/sanctuary-js/sanctuary-def#GlobalRegExp

Reply:
No idea

Reply:
No worries. This is not important. :)